Simulation Code/Base/Base.ino

```diff
@@ -1,5 +1,5 @@
 #include <Arduino.h>
-
+//Actual Revised Code
 // Define the pins for the water level sensors
```

Comment:
Correct me if I am wrong but I think we should group all the water level sensors together in a struct. So both the pin declarations and the variables that stores the water levels can be in the same struct.

Reply:
We should also rename any pin declarations from WaterLevelA to WaterPinA format so it is easier to tell between variables and pins